### DISCUSSION TITLE
[Expo] Can I use electric only with expo-sqlite and no pg db or the sync service?

### DISCUSSION THREAD

**Discussion by atanaskanchev** [electric-sql/electric#941] [Questions] [answered]
I'm building an offline first mobile app that as a MVP will work only on device and will not sync with a db. In the future I'm planning to sync with Supabase. So far I've tried watermelondb, but the dx is terrible. Couldn't wrap my head if I can achieve this in electric, any pointers?

**Comment by alco** [comment, marked as answer]
This has been answered [on Discord](https://discord.com/channels/933657521581858818/933657523049885698/1206232462708051968).

We have a local-only-first mode in the CLI API - https://electric-sql.com/docs/api/cli#local-only-first-mode. There are also upcoming changes to the `electrify()` TypeScript API that will make the auth token optional, allowing you to start with a local app and think about syncing with the server at a later point in development.

**Reply by atanaskanchev** [reply]
Thanks, I've missed that, will give it a try!
